inference-engine/tests/functional/plugin/shared/include/behavior/ov_config.hpp

```diff
@@ -0,0 +1,245 @@
+//         // Skip test according to plugin specific disabledTestPatterns() (if any)
+//         SKIP_IF_CURRENT_TEST_IS_DISABLED()
+//         // Create CNNNetwork from ngrpah::Function
+//         InferenceEngine::CNNNetwork cnnNet(function);
+//         std::map<std::string, std::string> config;
+//         ASSERT_NO_THROW(ie->get_metric(targetDevice, ov::cfg::supported_config_keys));
+//         ASSERT_NO_THROW(ie->compile_model(function, targetDevice, config));
+//     }
+
+//     using OVCorrectSingleOptionDefaultValueConfigTests = BehaviorTestsUtils::BehaviorTestsSingleOptionDefault;
+
+//     TEST_P(OVCorrectSingleOptionDefaultValueConfigTests, CheckDefaultValueOfConfig) {
+//         // Skip test according to plugin specific disabledTestPatterns() (if any)
+//         SKIP_IF_CURRENT_TEST_IS_DISABLED()
+//         // Create CNNNetwork from ngrpah::Function
+//         InferenceEngine::CNNNetwork cnnNet(function);
+//         ASSERT_NO_THROW(ie->get_metric(targetDevice, ov::cfg::supported_config_keys));
+//         ASSERT_EQ(ie->get_config(targetDevice, key), value);
+//     }
+
+//     using OVCorrectConfigTests = BehaviorTestsUtils::BehaviorTestsBasic;
+
+//     // Setting correct config doesn't throw
+//     TEST_P(OVCorrectConfigTests, SetCorrectConfig) {
+//         // Skip test according to plugin specific disabledTestPatterns() (if any)
+//         SKIP_IF_CURRENT_TEST_IS_DISABLED()
+//         // Create CNNNetwork from ngrpah::Function
+//         InferenceEngine::CNNNetwork cnnNet(function);
+//         ASSERT_NO_THROW(ie->get_metric(targetDevice, ov::cfg::supported_config_keys));
+//         ASSERT_NO_THROW(ie->set_config(configuration, targetDevice));
+//     }
+
+//     TEST_P(OVCorrectConfigTests, CanCompileModelWithCorrectConfig) {
+//         // Skip test according to plugin specific disabledTestPatterns() (if any)
+//         SKIP_IF_CURRENT_TEST_IS_DISABLED()
+//         // Create CNNNetwork from ngrpah::Function
+//         InferenceEngine::CNNNetwork cnnNet(function);
+//         ASSERT_NO_THROW(ie->compile_model(cnnNet, targetDevice, configuration));
+//     }
+
+//     TEST_P(OVCorrectConfigTests, CanUseCache) {
+//         // Skip test according to plugin specific disabledTestPatterns() (if any)
+//         SKIP_IF_CURRENT_TEST_IS_DISABLED()
+//         // Create CNNNetwork from ngrpah::Function
+//         InferenceEngine::CNNNetwork cnnNet(function);
+//         ie->set_config({ov::cfg::cache_dir("./test_cache")});
+//         ASSERT_NO_THROW(ie->compile_model(cnnNet, targetDevice, configuration));
+//         ASSERT_NO_THROW(ie->compile_model(cnnNet, targetDevice, configuration));
+//         CommonTestUtils::removeDir("./test_cache");
+//     }
+
+//     using OVCorrectSingleOptionCustomValueConfigTests = BehaviorTestsUtils::BehaviorTestsSingleOptionCustom;
+
+//     TEST_P(OVCorrectSingleOptionCustomValueConfigTests, CheckCustomValueOfConfig) {
```

Comment:
I think these tests should be changed

Reply:
Was waiting for tests infrastructure stabilization